tests/plugins/test_resolve_composes.py

```diff
@@ -1118,6 +1118,55 @@ def test_only_pulp_repos(self, workflow):
             .and_return(ODCS_COMPOSE))
         self.run_plugin_with_args(workflow)
 
+    @pytest.mark.parametrize(('content_sets', 'build_only_content_sets'), (
+        (True, True),
+        (True, False),
+        (False, True),
+        (False, False),
+    ))
+    def test_only_content_sets(self, workflow, content_sets, build_only_content_sets):
+        main_cs_list = ['pulp-spam-rpms', 'pulp-bacon-rpms', 'pulp-eggs-rpms']
+        dev_cs_list = ['dev-spam-rpms', 'dev-bacon-rpms', 'dev-eggs-rpms', 'pulp-spam-rpms']
+
+        if content_sets:
+            cs_json = {'x86_64': main_cs_list}
+            workflow._tmpdir.join('content_sets.yml').write(yaml.safe_dump(cs_json))
+
+        containter_json = {'compose': {'pulp_repos': True}}
+        if build_only_content_sets:
+            containter_json['compose']['build_only_content_sets'] = {'x86_64': dev_cs_list}
+        else:
+            containter_json['compose']['build_only_content_sets'] = None
+
+        workflow._tmpdir.join('container.yaml').write(yaml.safe_dump(containter_json))
```

Comment:
Sorry for my nit-pick :) How about:

```python
workflow._tmpdir.join('container.yaml').write(yaml.safe_dump({
    'compose': {
        'pulp_repos': True,
        'build_only_content_sets': {'x86_64': dev_cs_list} if build_only_content_sets else None
    }
})
```

Reply:
I'd rather keep simpler formatting as it is now, it is more readable